package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "start": "nodemon server/server.js",
-    "dev": "concurrently 'webpack serve --open' 'nodemon server/server.js'",
+    "dev": "concurrently 'webpack serve --open' 'nodemon server/server.js' ",
```

Comment:
Nitpicking here

```suggestion
    "dev": "concurrently 'webpack serve --open' 'nodemon server/server.js'",
```

Reply:
made this change locally. Will add to a future commit.